target/linux/ipq40xx/base-files/etc/board.d/01_leds

```diff
@@ -56,6 +56,10 @@ engenius,ens620ext)
 glinet,gl-ap1300)
 	ucidef_set_led_netdev "wan" "WAN" "green:wan" "eth1"
 	;;
+linksys,whw03v2)
+    ucidef_set_led_default "power" "POWER" "pca963x:blue" "1"
```

Comment:
This looks like ideal candidate for using alias in the DTS and setting the default to on there